src/transform/ThemeTransform.js

```diff
@@ -78,6 +78,41 @@ const classifyElements = element => {
     .forEach(div => {
       div.classList.add(CONSTRAINT.DIV_DO_NOT_APPLY_BASELINE)
```

Comment:
What do you think about consolidating all these selector and just running this function once? Something like:

```js
const selector = [
  'div.color_swatch div',
  'div[style*="position: absolute"]',
  'div[role="img"] div',
  ...,
  ...,
  ...
].join()
Polyfill.querySelectorAll(element, selector).forEach(element =>
  element.classList.add(CONSTRAINT.DIV_DO_NOT_APPLY_BASELINE))
```

Reply:
Sure, that sounds great, thanks @niedzielski!